pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[tool.poetry]
+name = "pygui"
+version = "0.1.0"
+description = "Reactive rendering for Python"
+authors = ["Korijn van Golen <korijn@gmail.com>", "Berend Klein Haneveld <berendkleinhaneveld@gmail.com>"]
+license = "MIT"
+homepage = "https://github.com/fork-tongue/pygui"
+readme = "README.md"
+
+[tool.poetry.extras]
+gfx = ["pygfx"]
+pyside = ["PySide6"]
+
+[tool.poetry.dependencies]
+python = ">=3.8"
+# NOTE: When observ is updated, the python marker can be removed
+observ = { version = "~0.9.0", python = "<3.11" }
+pygfx = { git = "https://github.com/pygfx/pygfx.git", rev = "main", optional = true }
```

Comment:
Should we require a release before merging to master?

Reply:
Yes, let's do that, otherwise it will just be weird. There is also another small thing which I might want to improve, so I'll open a PR for that ASAP.

Reply:
Actually, when writing the comment I thought you were talking about `observ`... Let me check if the example works with the last released version (I had some problems with the `Points` object, but I removed that from the example anyways...)

Reply:
Ah, yes, we need a release, to be able to leverage the keyword `before` in `WorldObject.add`.

Reply:
I just released 0.1.7 so I'll update the dependency and resolve this conversation.